contrib/orc/orc_format/orc_format.c

```diff
@@ -0,0 +1 @@
+//this is a test file for compiling.
```

Comment:
if this file is useless, please do not add it to the commit.
if you use it as a placeholder, please add a more formal c source file with license header, otherwise your file won't pass apache-rat-check and no one will merge your commit to master.

Reply:
Thanks for the information, I'll add the license header on the amending commits.